Tribler/community/gigachannel/community.py

```diff
@@ -186,18 +213,18 @@ def on_search_request(self, peer, request):
         metadata_type = {
             "": [REGULAR_TORRENT, CHANNEL_TORRENT],
             "channel": CHANNEL_TORRENT,
-            "torrent": REGULAR_TORRENT
+            "torrent": REGULAR_TORRENT,
         }.get(request.metadata_type.decode('utf8'), REGULAR_TORRENT)
 
         request_dict = {
             "first": 1,
             "last": max_entries,
             "sort_by": request.sort_by.decode('utf8'),
-            "sort_asc": request.sort_asc,
+            "sort_desc": request.sort_desc,
```

Comment:
What's the motivation behind changing the `sort_asc` to `sort_desc`?

Reply:
In SQL, by default stuff is sorted ascending. The same is true for the GUI. There is a modifier in ORM that makes the sorting key sort descending, like this: `desc(sort_expression)`. This change aligns the REST data format with the database commands. It simplifies the logic in between, by removing the unneccesary conversions.

Reply:
Ok.